packages/hydrojudge/src/judge/hack.ts

```diff
@@ -54,17 +58,39 @@ export async function judge(ctx: Context) {
         } else if (memory > ctx.config.memory * 1024) {
             status = STATUS.STATUS_MEMORY_LIMIT_EXCEEDED;
         } else {
-            ({ status, message } = await checkers[ctx.config.checker_type]({
-                execute: checker.execute,
-                copyIn: checker.copyIn || {},
-                input: { src: input },
-                output: { content: '' },
-                user_stdout: res.fileIds.stdout ? { fileId: res.fileIds.stdout } : { content: '' },
-                user_stderr: { fileId: res.fileIds.stderr },
-                score: 100,
-                detail: ctx.config.detail ?? true,
-                env: { ...ctx.env, HYDRO_TESTCASE: '0' },
-            }));
+            const stdResult = ctx.config.std ? await runQueued(
+                std.execute,
+                {
+                    stdin: { src: input },
+                    copyIn: std.copyIn,
+                    time: parseTimeMS(ctx.config.time || '1s'),
+                    memory: parseMemoryMB(ctx.config.memory || '256m'),
+                    cacheStdoutAndStderr: true,
+                },
+            ) : { status: STATUS.STATUS_ACCEPTED, fileIds: { stdout: null } };
+            const tmp = path.join(tmpdir(), `${ctx.request.rid}.out`);
+            ctx.clean.push(() => {
+                if (fs.existsSync(tmp)) fs.removeSync(tmp);
+                return res.fileIds['stdout'] ? client.deleteFile(res.fileIds['stdout']) : Promise.resolve();
+            });
+            if (stdResult.status === STATUS.STATUS_ACCEPTED) {
```

Comment:
what if std crashed?

Reply:
now it will return `STATUS.STATUS_SYSTEM_ERROR`